gc/base/MarkedObjectPopulator.cpp

```diff
@@ -72,9 +72,9 @@ MM_MarkedObjectPopulator::populateObjectHeapBufferedIteratorCache(omrobjectptr_t
 		} else {
 			/* set startPtr to just past the last object seen */
 #if defined(OMR_GC_MINIMUM_OBJECT_SIZE)
-			state->data1 = ((uintptr_t)object) + J9_GC_MINIMUM_OBJECT_SIZE;
+			state->data1 = ((uintptr_t)object) + OMR_MINIMUM_OBJECT_SIZE;
 #else /* OMR_GC_MINIMUM_OBJECT_SIZE */
-			state->data1 = ((uintptr_t)object) + sizeof(J9Object);
+			state->data1 = ((uintptr_t)object) + sizeof(fomrobject_t);
```

Comment:
I believe this is incorrect.  I agree that sizeof(J9Object) is wrong and would result in a compile failures if OMR_GC_MINIMUM_OBJECT_SIZE is not defined.  This needs to move the pointer past the last object.  The macro sizeof(fomrobject_t) is just going to move it past either 32 or 64 bits which is likely not the minimum size of an object.